### PR TITLE
feat(tutorial): complete step 4

### PR DESCRIPTION
**Pull request by shuyanh**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/components/Info/_info.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.info-section__heading {
+  @include carbon--type-style('heading-01');
+}
+.info-card {
+  margin-top: $spacing-09;
+  display: flex;
+  flex-direction: column;
+}
+
+swg {
```
Comment (kellychurchill) [inline src/components/Info/_info.scss]: This should be `svg`

File: src/components/Info/_info.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.info-section__heading {
+  @include carbon--type-style('heading-01');
+}
+.info-card {
+  margin-top: $spacing-09;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (kellychurchill) [inline src/components/Info/_info.scss]: This closing bracket should be after the `@include carbon--breakpoint(lg)` class

Comment (shuyanh) [reply src/components/Info/_info.scss]: updated per request

File: src/components/Info/_info.scss (hunk 1 above)
Comment (shuyanh) [reply src/components/Info/_info.scss]: updated per request

File: src/components/Info/_info.scss (hunk 2 above)
Comment (kellychurchill) [reply src/components/Info/_info.scss]: This closing bracket needs to be removed to resolve the grey left border that is showing up for all your divs (see your preview: https://deploy-preview-2079--carbon-tutorial.netlify.com/#/)

**Comment by shuyanh** [comment]
ha, now I see... thanks for reviewing, now corrected. the side grey bar is gone after correction.

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**